builtin.py

```diff
@@ -58,6 +59,34 @@ def advance(self, index: int) -> Node:
     def __str__(self):
         return self.title
 
+class Obj(Node):
+    def __init__(self, title:str, desc:str, player:Player):
+        super().__init__(title, desc)
+        self.desc = desc
+        self.player = player
+        if title.isnumeric(): #initializing money
+            self.amount_Money = int(title)
+        elif title[:2] == "po": #if first 2 characters are po then its a potion
```

Comment:
Instead of using conditionals to initialize the type of "Obj", could you make more classes inheriting this class? I think splitting up each class into explicit functionality (with one given input and output) will help the design of the project

Reply:
So, having a class for Potion and a class for Elixir, etc.

Reply:
Split the class initialization of different Items.